src/main/java/racinggame/domain/racing/RoundResult.java

```diff
@@ -1,20 +1,29 @@
 package racinggame.domain.racing;
 
-import racinggame.domain.car.Car;
-
 import java.util.ArrayList;
 import java.util.List;
 
 public class RoundResult {
-    private List<Car> result = new ArrayList<>();
+    private List<RacingCars> results;
+
+    public RoundResult() {
+        results = new ArrayList<>();
+    }
+
+    public RoundResult(List<RacingCars> racingCars) {
+        List<RacingCars> cars = new ArrayList<>();
+        cars.addAll(racingCars);
+        this.results = cars;
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        this.results = new ArrayList<>(cars);
```